endstate_correction/analysis.py

```diff
@@ -86,50 +89,84 @@ def plot_results_for_equilibrium_free_energy(
     plt.close()
 
 
-def plot_endstate_correction_results(
-    name: str, results: Results, filename: str = "plot.png"
-):
+def return_neq_correction(
+    results: Results, method: str = "NEQ", direction: str = "forw"
+) -> Tuple[float, float]:
+    # generate docstring
+    """Return the endstate correction for a given method and direction.
+    Args:
+        results (Results): instance of the Results class
+        method (str): NEQ or MBAR
```

Comment:
Also just a side note, but I think here it would be `NEQ or FEP`? But maybe I missed something

Reply:
corrected!